libs/components/inline-form/src/lib/modules/inline-form/types/inline-form-close-args.ts

```diff
@@ -1,7 +1,9 @@
 export interface SkyInlineFormCloseArgs {
   /**
-   * Returns a `string` value that explains why the inline form closed.
-   * The possible values are `cancel`, `delete`, `done`, and `save`.
+   * Returns a `string` value to explain why users clicked a custom button and initiated a `close` event.
+   * This correlates to the `action` in `SkyInlineFormButtonConfig`. The possible values are `cancel`, `delete`, `done`, `save`,
```

Comment:
Should we call out that these are pre-defined for the standard buttons that are not given via a `SkyInlineFormButtonConfig`? I don't pick that up from what I see here.

Reply:
I thought this interface was only for custom buttons. If so, do we really need to say anything about other buttons? I'm not arguing for or against. Just asking. ... If we want something, how about adding a `which is only necessary for custom buttons` to the end of the second sentence? (i.e., "This correlates to the the action in SkyInlineFormButtonConfig, which is only necessary for custom buttons."?